app/src/util/tools.ts

```diff
@@ -512,13 +512,104 @@ export const formatNumber = (number: string, decimals = 2): string => {
 }
 
+export const handleSmallShares = (
+  sharesAmount: string,
+  buyOrSell: string,
+  message: any,
+  balances: BalanceItem[],
+  outcomeIndex: number,
+) => {
+  if (Number(sharesAmount) < 0.01 && buyOrSell === 'buying') {
+    message(`Buying <${0.01} shares...`)
+  }
+
+  if (Number(sharesAmount) > 0.01 && buyOrSell === 'buying') {
+    message(`Buying ${sharesAmount} shares...`)
+  }
+
+  if (Number(sharesAmount) < 0.01 && buyOrSell === 'bought') {
+    message(`Successfully bought <${0.01} ${balances[outcomeIndex].outcomeName} shares`)
+  }
+
+  if (Number(sharesAmount) > 0.01 && buyOrSell === 'bought') {
+    message(`Successfully bought ${sharesAmount} ${balances[outcomeIndex].outcomeName} shares`)
+  }
+
+  if (Number(sharesAmount) < 0.01 && buyOrSell === 'sell') {
+    message(`Successfully sold <${0.01} ${balances[outcomeIndex].outcomeName} shares`)
+  }
+
+  if (Number(sharesAmount) > 0.01 && buyOrSell === 'sell') {
+    message(`Successfully sold ${sharesAmount} ${balances[outcomeIndex].outcomeName} shares`)
+  }
+
+  if (Number(sharesAmount) < 0.01 && buyOrSell === 'selling') {
+    message(`Selling <${0.01} shares...`)
+  }
+
+  if (Number(sharesAmount) > 0.01 && buyOrSell === 'selling') {
+    message(`Selling ${sharesAmount} shares...`)
+  }
+
+  if (Number(sharesAmount) < 0.01 && buyOrSell === 'sold') {
+    message(`Successfully sold <${0.01} shares`)
+  }
+
+  if (Number(sharesAmount) > 0.01 && buyOrSell === 'sold') {
+    message(`Successfully sold ${sharesAmount} shares`)
+  }
+}
+
+export const handleSmallDepositsAndWithdrawals = (
+  sharesAmount: string,
+  depositOrWithdraw: string,
+  message: any,
+  displayCollateral: Token,
+) => {
+  if (Number(sharesAmount) < 0.01 && depositOrWithdraw === 'depositing') {
+    message(`Depositing funds: <${0.01} ${displayCollateral.symbol}...`)
+  }
+
+  if (Number(sharesAmount) > 0.01 && depositOrWithdraw === 'depositing') {
+    message(`Depositing funds: ${sharesAmount} ${displayCollateral.symbol}...`)
+  }
+
+  if (Number(sharesAmount) < 0.01 && depositOrWithdraw === 'withdrawing') {
+    message(`Withdrawing funds: <${0.01} ${displayCollateral.symbol}...`)
+  }
+
+  if (Number(sharesAmount) > 0.01 && depositOrWithdraw === 'withdrawing') {
+    message(`Withdrawing funds: ${sharesAmount} ${displayCollateral.symbol}...`)
+  }
+
+  if (Number(sharesAmount) < 0.01 && depositOrWithdraw === 'hasDeposited') {
+    message(`Successfully deposited <${0.01} ${displayCollateral.symbol}`)
+  }
+
+  if (Number(sharesAmount) > 0.01 && depositOrWithdraw === 'hasDeposited') {
+    message(`Successfully deposited ${sharesAmount} ${displayCollateral.symbol}`)
+  }
+
+  if (Number(sharesAmount) < 0.01 && depositOrWithdraw === 'hasWithdrawn') {
+    message(`Successfully withdrew <${0.01} ${displayCollateral.symbol}`)
+  }
+
+  if (Number(sharesAmount) > 0.01 && depositOrWithdraw === 'hasWithdrawn') {
+    message(`Successfully withdrew ${sharesAmount} ${displayCollateral.symbol}`)
+  }
+}
+
```

Comment:
Shit man seems that you misunderstood my feedback or that i didn't communicate it quite well....So i didn't mean to put every message in the tools file. But to just take logic into one function like this:
`function smallAmoutChecker(value){
 if(value<0.01)return '<0.01' 
else return value
}
`
And then for example instead of 
setMessage(`Successfully sold ${sharesAmount} '${balances[outcomeIndex].outcomeName}' shares.`)
you would do
setMessage(`Successfully sold ${smallAmountChecker(sharesAmount)} '${balances[outcomeIndex].outcomeName}' shares.`)
Does that make sense?

Reply:
yes, sorry. misunderstanding. i thought you wanted all of the logic inside tools to reduce noise in other files...